_html/scss/layout/_grid.scss

```diff
@@ -47,3 +47,15 @@
         width: 33.3333%;
     }
 }
+
+.l-col-4 {
+    width: 100%;
+
+    @include tablet {
+        width: 50%;
```

Comment:
Hmm, this doesn't seem ok, because l-col-2 already has 50%. How about we do some Bootstrapish grid?
Keep l-col-1 and l-col-2 and rename them l-col-t-1, l-col-t-2 (t for tablet). And add l-col-md-4 (md for medium desktop). In this way you could add both classes to an element, like `<div class='l-col-t-2 l-col-md-4'></div>`and achieve the same (have them 25% for medium and 50% for tablet). 
Do you think this could work?